docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```

Comment:
Thoughts on using `npm` instead of `yarn` - just for consistency across our codebases?

Reply:
Yeah, I've been using npm to deploy it, this is just a readme created by the template. I can update it.